packages/api/src/controllers/helpers.js

```diff
@@ -163,6 +164,32 @@ export async function sendgridEmail({
   await SendgridMail.send(msg);
 }
 
+export function sendgridValidateEmail(email, sendgridApiKey) {
+  SendgridClient.setApiKey(sendgridApiKey);
+
+  const request = {
+    url: `/v3/validations/email`,
+    method: "POST",
+    body: { email, source: "signup" },
+  };
+  client
```

Comment:
typo here? should it be `SendgridClient`?

Reply:
Ugh yes, thanks for spotting that. I hate JS, doesn't warn even on a typo 🙄

Reply:
Couldn't hold myself. Converted these helpers to TypeScript 😡

Reply:
:rofl: I was going to suggest maybe make it a module? `helpers/sendgrid.ts` etc?

Reply:
I think we can do that if we end up keeping this email validation logic. Otherwise, it might be just 1 function in there for sending emails.